src/components/tag/Tag.module.css

```diff
@@ -1,20 +1,25 @@
 :root {
   --conduction-tag-font-size: var(--skeleton-font-size-sm);
   --conduction-tag-color: var(--skeleton-black);
-  --conduction-tag-backgrond-color: var(--skeleton-color-grey-1);
+  --conduction-tag-background-color: var(--skeleton-color-grey-1);
   --conduction-tag-border-radius: var(--skeleton-border-radius-md);
   --conduction-tag-padding: var(--skeleton-size-xs);
+  --conduction-tag-margin: var(--skeleton-size-2xs);
```

Comment:
The name of this design token may be more descriptive. Right now, when implementing the `Tag` component with this token, it's unclear _what_ this margin does.

In context it's clear, it creates a margin between the icon and the label.

Maybe something like `--conduction-tag-child-margin-inline`?